specifyweb/specify/load_datamodel.py

```diff
@@ -39,7 +40,7 @@ def get_table_strict(self, tablename: str) -> 'Table':
         for table in self.tables:
             if table.name.lower() == tablename:
                 return table
-        raise TableDoesNotExistError("No table with name: %r" % tablename)
+        raise TableDoesNotExistError(_("No table with name: %(table_name)r") % {'table_name':tablename})
```

Comment:
I'm not sure it makes sense to add translations for these types of errors. They indicate programming errors rather than user errors.

Reply:
I see. I wasn't sure which exceptions were user-facing and which weren't. (e.x, exceptions in the WB are shown to the user as validation messages)

Reply:
Do you know offhand if there are using-facing exceptions in any other files? If not, I can revert those files

Reply:
I didn't see any other non-user exceptions in the translations table. As far as exceptions users will see, the only ones that come to mind are the business rules.

Reply:
Un-escaped strings in load_datamodel.py in d3e9e24